dataland-community-manager/src/main/kotlin/org/dataland/datalandcommunitymanager/repositories/utils/TemporaryTables.kt

```diff
@@ -23,19 +23,27 @@ class TemporaryTables private constructor() {
             "FROM data_requests d " +
             "JOIN status_table ON status_table.request_id = d.data_request_id " +
             "WHERE " +
-            "(:#{#searchFilter.dataTypeFilterLength} = 0 " +
-            "OR d.data_type = :#{#searchFilter.dataTypeFilter}) AND " +
-            "(:#{#searchFilter.userIdFilterLength} = 0 " +
-            "OR d.user_Id = :#{#searchFilter.userIdFilter}) AND " +
-            "(:#{#searchFilter.requestStatusLength} = 0 " +
-            "OR status_table.request_status = :#{#searchFilter.requestStatus} ) AND " +
-            "(:#{#searchFilter.accessStatusLength} = 0 " +
-            "OR status_table.access_status = :#{#searchFilter.accessStatus}  ) AND " +
-            "(:#{#searchFilter.reportingPeriodFilterLength} = 0 " +
-            "OR d.reporting_period = :#{#searchFilter.reportingPeriodFilter}) AND " +
-            "(:#{#searchFilter.datalandCompanyIdFilterLength} = 0 " +
-            "OR d.dataland_company_id = :#{#searchFilter.datalandCompanyIdFilter}) " +
+            "(:#{#searchFilter.shouldFilterByDataType} = false " +
+            "OR d.data_type IN :#{#searchFilter.usedDataTypeFilter}) AND " +
```

Comment:
In my opinion the prefix "used" does not add any value to the variable name, i would be in favour to not use it.

Reply:
For me they are not necessary. The searchfilter."variable" is understandable.

Reply:
ok